apps/src/StudioApp.js

```diff
@@ -1629,7 +1629,9 @@ StudioApp.prototype.runButtonClickWrapper = function (callback) {
  */
 StudioApp.prototype.configureDom = function (config) {
   var container = document.getElementById(config.containerId);
-  container.innerHTML = config.html;
+  if (typeof config.html !== 'undefined') {
+    container.innerHTML = config.html;
+  }
```

Comment:
Thanks for explaining, it was not clear that this conditional is based on whether this app is using react to render. I think it would suffice to add a comment noting that `config.html` will be empty for apps which are rendering via React.